components/brave_wallet_ui/common/api/hardware_keyrings.ts

```diff
@@ -7,17 +7,31 @@ import { assert } from 'chrome://resources/js/assert.m.js'
 import { BraveWallet } from '../../constants/types'
 import LedgerBridgeKeyring from '../../common/hardware/ledgerjs/eth_ledger_bridge_keyring'
 import TrezorBridgeKeyring from '../../common/hardware/trezor/trezor_bridge_keyring'
-
+import { FilecoinKeyring, LedgerKeyring } from '../hardware/hardwareKeyring'
+import FilecoinLedgerKeyring from '../hardware/ledgerjs/filecoin_ledger_keyring'
 export type HardwareKeyring = LedgerBridgeKeyring | TrezorBridgeKeyring
 
-const VendorTypes = [
-  BraveWallet.TREZOR_HARDWARE_VENDOR,
-  BraveWallet.LEDGER_HARDWARE_VENDOR
-] as const
+export enum HardwareCoins {
+  FILECOIN = 'f',
+  ETH = 'e'
```

Comment:
Can we use more descriptive names here? Ex: `filecoin` and `ethereum`.

Reply:
done